tests/config/test_base_configuration.py

```diff
@@ -0,0 +1,188 @@
+import json
+import os
+import shutil
+
+import pytest
+
+from nucypher.config.base import BaseConfiguration
+from nucypher.config.constants import DEFAULT_CONFIG_ROOT
+
+#
+# Local Test Constants
+#
+
+configuration_name = 'something'
+expected_extension = 'json'
+configuration_value = 're-emerging llamas'
+modifier = '1'
+manual_expected_default_filepath = os.path.join('/', 'tmp', 'something.json')
```

Comment:
No Windows support I guess 😅

Reply:
This is within a test, but Yeah.... Perhaps we need an Issue for windows paths support.